packages/rocketstyle/src/__stories__/Button.tsx

```diff
@@ -30,8 +30,11 @@ const RightContent = config.styled.span`
     )
   }};
 `
+type Props = {
+  active: boolean
+}
 
-export const Button = rocketstyle()({ name: 'Button', component: Element })
+export const Button= rocketstyle<Props>()({ name: 'Button', component: Element })
```

Comment:
U can also specify generic **basetype**  , but must be Cyclic type like the default